quickevent/app/quickevent/plugins/CardReader/CardReader.pri

```diff
@@ -0,0 +1,32 @@
+message(including plugin $$PWD)
+
+PLUGIN_NAME = CardReader
+
+include (src/src.pri)
+
+RESOURCES += \
+      $$PWD/$${PLUGIN_NAME}.qrc
+
+OTHER_FILES += \
+        $$PWD/qml/reports/* \
+
+TRANSLATIONS += \
+        $$PWD/$${PLUGIN_NAME}.cs_CZ.ts \
```

Comment:
I'm afraid that we have to merge also localization into one file per language somehow

Reply:
this is not needed, as for now we do not use qmake to build the language files.

There is a possibility to move the language file generation into the qmake, but as for now we use a special parts of the build scripts for that.

Reply:
the `TRANSLATIONS` variable has been actually redundant all the time.
In a new issue we can discuss handling of the translations, its possible to simplify it, it will take some time though. using `CONFIG += lrelease embed_translations`, changing the .ts files naming style (multiple dots in the name cause problems), refactor translation loading and possibly merging the files into one,..